www/heatmap.html

```diff
@@ -0,0 +1,236 @@
+                    <form method="get" action="./data/heatmap-tutorial.zip">
+                        <button class="btn btn-primary btn-lg text-center" type="submit"><span class="glyphicon glyphicon-cloud-download" aria-hidden="true"></span> Download Heatmap Tutorial Files</button>
+                    </form>
+                </div>
+                <div class="col-lg-4">
+                    <form method="get" action="./heatmap-demo.html">
+                        <button class="btn btn-primary btn-lg text-center" type="submit">Demo</button>
+                    </form>
+                </div>
+            </div>
+            <br>
+
+            <div id="heatmapContainer" style="margin: auto; width: 100%; height: auto; max-width: 700px">
+                    <img id="webgazer-pic" src="./media/heatmap-demo.png" style="width: 100%; height: auto; max-width: 700px">
+            </div>
+        </div>
+            <script src="./js/heatmap.js"></script>
+            <script src="https://code.jquery.com/jquery-1.11.0.min.js"></script>
+            <script src="./js/heatmap-data.js"></script>
+            <script src="./js/heatmap-webgazer.js"></script>
+
+        <div id="heatmap-usage" class="container pad">
+            <hr class="featurette-divider">
+            <h2 class="text-center">Usage</h2>
+            <br>Begin by creating a div element that will house all of the heatmap's visual rendering on your page.
+            
+<pre><code class="html">&lt;!-- Heatmap Container --&gt;
+&lt;div id="heatmapContainer">&lt;div&gt;</code></pre>
+
+            <p><br>Create HTML script elements that import the javascript files needed for the visualization. The script element needs to be given the location that the <code class="javascript">heatmap.js</code> file is stored. <b>Note:</b> The period in <code class="javascript">"./js/heatmap.js"</code> represents the current directory.</p>
+
+<pre><code class="html">&lt;!-- Script Element --&gt;
+&lt;script src="./js/heatmap.js"&gt;&lt;script&gt; 
+</code></pre>
+
+<p><br>Initialize a heatmap instance with a few lines of javascript.</p>
+<pre><code class="javascript hljs">var heatmap = h337.create({
+    container: document.getElementById('heatmapContainer') // refers to the container that you created above
+});
+</code></pre>
+
+            <p><code class="javascript">Heatmap.js</code> supports many kinds of visual configurations that change the heatmaps appearance. View these settings in the initialization <a href="https://www.patrick-wied.at/static/heatmapjs/docs.html">documentation.</a><p>
+            <p><br>Calling the <code class="javascript">setData()</code> function on the <code class="javascript">heatmap</code> variable: <code class="javascript">heatmap.setData(data)</code> allows the user to provide the heatmap instance with data to display. The data has to be in JSON format, below is an example of how the information should be structured.</p>
+
+<pre><code class="javascript hljs">{
```

Comment:
As I said before - is it necessary to repeat this when it's copied again below just as a variable instead?